src/components/EventCheckIn/index.jsx

```diff
@@ -20,7 +23,14 @@ const EventCheck = props => {
         <Button onClick={props.onSubmit} className="submit">
           Submit
         </Button>
+
       </div>
+      { props.user.profile.accountType === "STAFF" &&
+        <div className="staff-checkin-div">
+          <label className="staff-check-in-label">Staff Check In</label>
+          <Checkbox onChange={props.onAsStaffChange}/>
+        </div>
+      }
```

Comment:
not sure if the current format is convention but I think this block would be a little more obvious/readable as a conditional

Reply:
This is ok, would use a conditional if there was something to display if accountType wasn't STAFF but there isn't anything to display.